CODEOWNERS

```diff
@@ -7,3 +7,6 @@
 # Prevent PRs that only modify dependencies from notifying everyone
 package.json @ghost
 yarn.lock @ghost
+
+# Owners for the Playwright end-to-end test directory
+playwright/ @coveo/search-qa
```

Comment:
I get the error "Unknown owner: make sure the team @coveo-platform/qa exists, is publicly visible, and has write access to the repository"
...but this looks right: https://github.com/orgs/coveo/teams/search-qa

Reply:
Write access granted!